engines/sci/parser/vocabulary.cpp

```diff
@@ -188,6 +188,45 @@ bool Vocabulary::loadParserWords() {
 		seeker += 3;
 	}
 
+	// Additional translated words
```

Comment:
This should be broken out into its own function with more comments explaining what it does, where the format comes from, everything. Right now, a reader wouldn't know that it was for fan translations or even what language it applies to. This is a critical reference function that's already quite big; a third party extra like this should be separated. "What actually uses this code?" is the question we're always asking.

Reply:
Moved to own function with comments